Shop/src/main/resources/templates/home.html

```diff
@@ -18,12 +21,17 @@ <h2>Items List:</h2>
             <th>Description</th>
             <th>Price</th>
             <th>Quantity</th>
+            <th>Action</th>
         </tr>
         <tr th:each="item : ${items}">
             <td th:text="${item.name}"></td>
             <td th:text="${item.description}"></td>
             <td th:text="${item.price}"></td>
             <td th:text="${item.quantity}"></td>
+            <td th:if="${item.quantity > 0}"><a th:href="@{'/shop/item/sell/'+${item.id}}"
```

Comment:
vietoj to paties elemento perrasymo galetum pabandyti tik elemento dalis itakoti, pvz `th:class="${item.quantity > 0}? 'btn btn-primary' : 'btn btn-primary disabled'"`